build.gradle

```diff
@@ -25,13 +25,13 @@ def getSqlFile(def email) {
 }
 
 def update() {
-        def sql = Sql.newInstance("jdbc:mysql://localhost:3306/", "root", "1111", "com.mysql.jdbc.Driver")
+        def sql = Sql.newInstance("jdbc:mysql://localhost:3306/", "root", "", "com.mysql.jdbc.Driver")
```

Comment:
maybe it will be better use some config file for db_user, db_pass, db_url like we have in spring-contex.xml for data.source.